### PR TITLE
Use `-` for Markdown slugs instead of `_`

### PR DESCRIPTION
**Pull request by zwaldowski**
See #524.

Requires spec updates. Workin' on it…

### CODE REVIEW

**Comment by jpsim** [comment]
Looking good! Please update the integration specs submodule to point to the latest `master`, CI will run again and then this should be good to merge!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## Master
+
+##### Bug Fixes
+
+* Uses GitHub-Flavored Markdown syntax for anchors when rendering README pages.
```
Comment (jpsim) [inline CHANGELOG.md]: 2 trailing spaces on the last line describing the change please. (from https://github.com/realm/jazzy/blob/master/CONTRIBUTING.md)

**Comment by zwaldowski** [comment]
Updated! Let 👷 give it the go-ahead!

**Comment by jpsim** [comment]
¡Muy bien!

**Pull request merged**